packages/components/src/components/SquiggleErrorAlert.tsx

```diff
@@ -35,24 +38,28 @@ const WithHeader: FC<PropsWithChildren<{ header: string }>> = ({
   children,
 }) => (
   <div>
-    <div className="font-medium">{header}</div>
-    <div className="ml-4">{children}</div>
+    <div className="text-sm font-medium mb-1 text-gray-700">{header}</div>
+    <div className="ml-1">{children}</div>
   </div>
 );
 
 const StackTraceFrame: FC<{ frame: SqFrame }> = ({ frame }) => {
   const location = frame.location();
+  const name = frame.name();
   return (
-    <div>
-      {frame.name()} {location ? <LocationLine location={location} /> : ""}
+    <div className="grid grid-cols-[1fr_3fr] gap-x-2">
```

Comment:
`1fr_3fr` means that the distance between columns will be different depending on screen size, and it might become too big on large screens:

<img width="721" alt="Captura de pantalla 2023-12-05 a la(s) 13 35 17" src="https://github.com/quantified-uncertainty/squiggle/assets/89368/9cab620b-a59e-426f-bdf5-5107b1023c8e">

<img width="1248" alt="Captura de pantalla 2023-12-05 a la(s) 13 35 22" src="https://github.com/quantified-uncertainty/squiggle/assets/89368/e14a7535-d768-4c60-844c-b563b187f579">

I'd suggest fixed left column instead, or something like `minmax(40px, max-content)` and a slightly larger gap if there's a risk that left column will be too small when function names are small.

(This will also force you to move `grid` to the parent component and return a fragment from this component)

Reply:
Good call